views/js/qtiCreator/widgets/interactions/mediaInteraction/states/Question.js

```diff
@@ -237,6 +238,113 @@ define([
             }
         };
 
+        const setUpCallbacks = function setUpCallbacks() {
+            //init data change callbacks
+            const callbacks = {
+                autostart: function autostart(boundInteraction, attrValue, attrName) {
+                    interaction.attr(attrName, attrValue);
+                    if (attrValue === false) {
+                        // reset subpanel properties to defaults
+                        interaction.removeAttr('data-autostart-delay-ms');
+                        interaction.removeClass('hide-player');
+                        $container.removeClass('dimmed');
+                        interaction.removeClass('sequential');
+                    }
+                    renderForm();
+                    reRender();
```

Comment:
I had to add back one `reRender()` - without it, the mediaEditor is never recreated because the `playerready` event doesn't fire.

Reply:
All right. Can we solve the blinking problem somehow as well?